content/swagger/resources/measurement_families/routes/measurement_families.yaml

```
@@ -163,3 +163,46 @@ get:
       $ref: "#/responses/403Error"
     406:
       $ref: "#/responses/406Error"
+patch:
+  summary: Update/create several measurement families
+  operationId: "patch_measurement_families"
+  tags:
+    - Measurement family
+  x-versions:
+    - "Serenity"
+  description: This endpoint allows you to update and/or create several measurement families at once.
+  x-body-by-line: Contains several lines, each line is a measurement family in JSON standard format
+  parameters:
+    - name: body
+      in: body
+      schema:
+        $ref: "#/definitions/MeasurementFamily"
+        x-examples:
+          x-example-1:
+            '{"code":"Illuminance","standard_unit_code":"LUX","units":[{"code": "LUX","convert_from_standard":[{"operator": "mul", "value": "1"}], "symbol": "lx"}]}'
+          x-example-2:
+            '{"code":"Binary","standard_unit_code":"BIT"}'
+          x-example-3:
+            '{"code":"Area","units":[{"code": "ARPENT","convert_from_standard":[{"operator": "mul", "value": "3418.89"}], "symbol": "arpent"}]}'
+  responses:
+    200:
+      description: OK
+      x-details: Returns a plain text response whose lines are JSON containing the status of each update or creation
+      x-no-entity: true
+      schema:
+        $ref: "#/definitions/ErrorByLine"
+      examples:
+        x-example-1:
+          '{"line":1,"code":"Illuminance","status_code":201}'
+        x-example-2:
+          '{"line":2,"code":"Binary","status_code":422,"message":"The standard unit code of the "Binary" measurement family cannot be changed"}'
```

```suggestion
          '{"line":2,"code":"Binary","status_code":422,"message":"The standard unit code of the \"Binary\" measurement family cannot be changed"}'
```
You need to escape the `"`. :)